packages/web-components/fast-foundation/src/tree-view/tree-view.ts

```diff
@@ -278,13 +284,13 @@ export class FASTTreeView extends FASTElement {
         if (this.currentFocused === null || !this.contains(this.currentFocused)) {
             this.currentFocused = this.getValidFocusableItem();
         }
-
+        console.log(this.currentFocused, "this current focused");
```

Comment:
Remove `console.log`?

Reply:
Yep, artifact of working at 2am. Removed downstream and will push the update when I can confirm all tests are passing.